.github/workflows/test.yml

```diff
@@ -64,46 +65,18 @@ jobs:
       # - phpcs_version: The PHPCS versions to test against.
       #   IMPORTANT: test runs shouldn't fail because of PHPCS being incompatible with a PHP version.
       #   - PHPCS will run without errors on PHP 5.4 - 7.4 on any supported version.
-      #   - PHP 8.0 needs PHPCS 3.5.7+ to run without errors.
+      #   - PHP 8.0 needs PHPCS 3.5.7+ to run without errors, and we require a higher minimum version.
+      #   - PHP 8.1 needs PHPCS 3.6.1+ to run without errors, but works best with 3.7.1+, and we require at least this minimum version.
       # - The `wpcs_version` key is added to allow additional test builds when multiple WPCS versions
       #   would be supported. As, at this time, only the latest stable release of WPCS is supported,
       #   no additional versions are included in the array.
       # - experimental: Whether the build is "allowed to fail".
       matrix:
-        php: ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
-        phpcs_version: ['3.5.5', 'dev-master']
+        php: ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+        phpcs_version: ['3.7.1', 'dev-master']
         wpcs_version: ['2.3.*']
         experimental: [false]
 
-        include:
-          # Complete the matrix by adding PHP 8.0, but only test against compatible PHPCS versions.
-          - php: '8.0'
-            phpcs_version: 'dev-master'
-            wpcs_version: '2.3.*'
-            experimental: false
-          - php: '8.0'
-            # PHPCS 3.5.7 is the lowest version of PHPCS which supports PHP 8.0.
-            phpcs_version: '3.5.7'
-            wpcs_version: '2.3.*'
-            experimental: false
-
-          # Complete the matrix by adding PHP 8.1, but only test against compatible PHPCS versions.
-          - php: '8.1'
-            phpcs_version: 'dev-master'
-            wpcs_version: '2.3.*'
-            experimental: false
-          - php: '8.1'
-            # PHPCS 3.6.1 is the lowest version of PHPCS which supports PHP 8.1.
-            phpcs_version: '3.6.1'
-            wpcs_version: '2.3.*'
-            experimental: false
-
-          # Experimental builds. These are allowed to fail.
-          #- php: '8.2'
-          #  phpcs_version: 'dev-master'
-          #  wpcs_version: '2.3.*'
-          #  experimental: true
-
```

Comment:
Just checking - you are now also removing the "placeholder" for the "future PHP" build - even though it's currently commented out. While this should probably be changed to `8.3` now (and WPCS `dev-develop`) and may potentially still fail, I'm not sure it should be removed.

(The other builds which are being removed are replaced by a build via the matrix, this one isn't being replaced in this PR.)

Reply:
Yes - I recall that 8.2 wasn't working correctly, though I can't recall why for the moment. It needs some separate testing, and this 8.2 and 8.3 can be added back in as non-comment as needed. Easy enough to find this placeholder again in Git :-) 

https://github.com/Automattic/VIP-Coding-Standards/issues/745 opened to look into this.